vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/order_handler.rb

```diff
@@ -15,6 +15,8 @@ def initialize(occupancy)
       end
 
       def process
+        return unless user_can_purchase_secure_room?
```

Comment:
This covers occupancies that we don't want orders for, provided someone who can enter for free can't also purchase the room. Otherwise, are there any error cases that we specifically want to watch out for?